tests/test_ssl.py

```diff
@@ -31,53 +11,37 @@ async def app(scope, receive, send):
     await send({"type": "http.response.body", "body": b"", "more_body": False})
 
 
-@pytest.mark.skipif(
-    sys.platform.startswith("win"), reason="Skipping SSL test on Windows"
-)
-def test_run(tls_ca_certificate_pem_path, tls_ca_certificate_private_key_path):
+@pytest.mark.asyncio
+async def test_run(tls_ca_certificate_pem_path, tls_ca_certificate_private_key_path):
     config = Config(
         app=app,
         loop="asyncio",
         limit_max_requests=1,
         ssl_keyfile=tls_ca_certificate_private_key_path,
         ssl_certfile=tls_ca_certificate_pem_path,
     )
-    server = Server(config=config)
-    thread = threading.Thread(target=server.run)
-    thread.start()
-    while not server.started:
-        time.sleep(0.01)
-    with no_ssl_verification():
-        response = requests.get("https://127.0.0.1:8000")
+    async with run_server(config):
+        async with httpx.AsyncClient(verify=False) as client:
```

Comment:
Note for the future: maybe Requests didn't support it, but now with HTTPX I think we could pass `verify=<ca_cert>`, rather than disabling SSL verification.

Reply:
I wanted to mimic the no verification, maybe another PR can use the certificates fixtures !